dashboards/lodestar_networking.json

```diff
@@ -768,6 +768,246 @@
       "title": "Topic peers count",
       "type": "timeseries"
     },
+    {
+      "collapsed": false,
+      "gridPos": {
+        "h": 1,
+        "w": 24,
+        "x": 0,
+        "y": 30
+      },
+      "id": 608,
+      "panels": [],
+      "title": "Network Worker",
+      "type": "row"
+    },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "${DS_PROMETHEUS}"
+      },
+      "description": "Async time to send a message across the worker/parent port",
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "palette-classic"
+          },
+          "custom": {
+            "axisCenteredZero": false,
+            "axisColorMode": "text",
+            "axisLabel": "",
+            "axisPlacement": "auto",
+            "barAlignment": 0,
+            "drawStyle": "line",
+            "fillOpacity": 23,
+            "gradientMode": "opacity",
+            "hideFrom": {
+              "graph": false,
+              "legend": false,
+              "tooltip": false,
+              "viz": false
+            },
+            "lineInterpolation": "linear",
+            "lineWidth": 1,
+            "pointSize": 5,
+            "scaleDistribution": {
+              "log": 2,
+              "type": "log"
+            },
+            "showPoints": "never",
+            "spanNulls": false,
+            "stacking": {
+              "group": "A",
+              "mode": "none"
+            },
+            "thresholdsStyle": {
+              "mode": "off"
+            }
+          },
+          "mappings": [],
+          "unit": "s"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 6,
+        "w": 12,
+        "x": 0,
+        "y": 31
+      },
+      "id": 148,
+      "options": {
+        "graph": {},
+        "legend": {
+          "calcs": [],
+          "displayMode": "list",
+          "placement": "bottom",
+          "showLegend": true
+        },
+        "tooltip": {
+          "mode": "multi",
+          "sort": "none"
+        }
+      },
+      "pluginVersion": "7.4.5",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_PROMETHEUS}"
+          },
+          "editorMode": "code",
+          "exemplar": false,
+          "expr": "(\n  sum(rate(\n    lodestar_network_worker_wire_events_on_worker_thread_latency_sum[$rate_interval]\n  )) \n  +\n  sum(rate(\n    lodestar_network_worker_wire_events_on_main_thread_latency_sum[$rate_interval]\n  ))\n)\n/\n(\n  sum(rate(\n    lodestar_network_worker_wire_events_on_worker_thread_latency_count[$rate_interval]\n  ))\n  +\n  sum(rate(\n    lodestar_network_worker_wire_events_on_main_thread_latency_count[$rate_interval]\n  ))\n)",
```

Comment:
@matthewkeil I found this metric from #5800, do you want it to be part of this PR too?

Reply:
I leave that up to you @tuyennhv.  I changed them around a few times so they may be a bit different from the last time you checked.  Feel free to either pull the updated ones into this PR or I can merge the additions from yours into mine.  Whichever works for me.  I leave the decision with you. @wemeetagain mentioned this to me as well so just let me know how to help because I think he wants to get these into 1.10

Here is a screenshot of the updated version I made.  I would go with your overall average because my calculation seems incorrect, but I kinda prefer the layout I settled on with one panel broken out by `eventName` and the other panel with just the three averages on it. `Average`, average of `workerToMain` and `mainToWorker` because they seem to be pretty different in the two directions and the individual metrics are also quite distinct in value for each event type so there will be a lot of opportunity to tune the individual events but they are hard to glance at to see if the averages is changing.

![Screenshot 2023-08-01 at 6 37 40 AM](https://github.com/ChainSafe/lodestar/assets/18608739/23214129-af0f-4b3f-8d1c-785c6d583afb)

![Screenshot 2023-08-01 at 6 37 47 AM](https://github.com/ChainSafe/lodestar/assets/18608739/78ddb602-4258-4016-a429-c32c01a026e2)

Reply:
ok then let's also add your panels to grafana, you can change them later @matthewkeil